### DISCUSSION TITLE
Monthly ESMValTool community meeting September 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#46] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next monthly ESMValTool community meeting will be in September. The result of the poll is that the meeting take place on

Thursday, September 8 at 4:00 PM CEST. 

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817?pwd=S2x5TE1HMlAxUEIzdTlsSzBUUTl3QT09).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/41#discussioncomment-3333157

**Comment by bouweandela** [comment]
Today's meeting will be hosted by @valeriupredoi. Thanks V!

**Reply by valeriupredoi** [reply]
indeed, I'll be your captain onboard ESMValTool Telco Flight 0922, and by the looks of it I'll be the flight attendant too, since I'll have to take minutes :airplane:

**Comment by valeriupredoi** [comment]
**Minutes from the meeting**

- Attendance: @hb326 @glpotter @schlunma @zklaus @remi-kazeroni @mo-tgeddes @catherinehardacre @KatherineTomkins @egalytska @valeriupredoi @LisaBock @bsolino 

**Shtuff**

- @LisaBock ->
  -  DLR folk will organize the EGU training/tutorial course on ESMValTool at EGU (duh!) - **if you wanna get involved, get in touch with Lisa** (I will myself)
- @hb326 ->
  - open an issue about reanalysis data replication, comparison and cmorization (w/ @glpotter )
  - CMIP7 coming soon: tasks list circulated, if you want to get involved you should apply - ask Birgit if any questions: https://www.wcrp-climate.org/wgcm-cmip/call-members-cmip7
- @catherinehardacre ->
  - station based data: OBS -ERRONET data from stations: do we need to bring in the data on JASMIN or we should leave it somewehere
  - @zklaus : used to be in v1, asking about Jane M. - Klaus and Catherine will get together on a call about this
  - Jerry: NASA reaching out for obs4mips data
- @glpotter ->
  - question about cmorization attributes
  - ERA-I data - natively cmorized? Comes almost cmorized
